service/application/src/main/java/com/solace/maas/ep/event/management/agent/service/ImportService.java

```diff
@@ -50,6 +50,7 @@ public ImportService(ProducerTemplate producerTemplate, DataCollectionFileServic
         this.eventPortalProperties = eventPortalProperties;
     }
 
+    @SuppressWarnings("PMD.CloseResource")
     public void importData(ImportRequestBO importRequestBO) throws IOException {
         boolean isEMAStandalone = eventPortalProperties.getGateway().getMessaging().isStandalone();
         InputStream importStream = importRequestBO.getDataFile().getInputStream();
```

Comment:
Same reason as above. It will cause the InputStream close early and fail the import.